lib/scss_lint/reporters/codeclimate_reporter.rb

```diff
@@ -24,7 +24,9 @@ def report_lints
             }
           }
         }.to_json
-      end.join("\0")
+        $stdout << "#{issue_json}\0"
```

Comment:
Trying to remember why I did it this way. I was either imitating something else in this engine or in scss-lint itself. Agreed it's unusual for our engines.